docs/index.rst

```diff
@@ -203,7 +206,7 @@ You can see more in the :ref:`persistence` chapter.
 Pre-built Faceted Search
 ------------------------
 
-If you have your ``DocType``\ s defined you can very easily create a faceted
+If you have your ``Doccument``\ s defined you can very easily create a faceted
```

Comment:
@HonzaKral, typo here, Doccument => Document

Reply:
thanks, fixed in 8ecf484